.azure-pipelines/mgmt.yml

```diff
@@ -0,0 +1,27 @@
+pr:
+  branches:
+    include:
+    - dev
+
+pool:
+  vmImage: 'VS2017-Win2016'
+
+steps:
+  - task: CmdLine@1
+    inputs:
+      filename: C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.6.2 Tools\sn.exe
+      arguments: -Vr *,31bf3856ad364e35
+  - task: MSBuild@1
+    inputs:
+      solution: build.proj
+  - task: MSBuild@1
+    inputs:
+      solution: build.proj
+      msbuildArguments: /t:RunTests /clp:ShowTimestamp /flp:Summary;Verbosity=minimal;LogFile=msbuild.sum /flp1:warningsonly;logfile=msbuild.wrn /flp2:errorsonly;logfile=msbuild.err /p:IgnorePathTokens="Microsoft.Rest.ClientRuntime.Tracing.Tests Microsoft.Azure.Services.AppAuthentication.IntegrationTests KeyVault\dataPlane LocationBasedServices Az.Auth.Tests BotService.Tests Search Batch.FileStaging.Tests Azure.Batch.IntegrationTests Azure.Batch.ProtocolTests Batch\Support\FileConventions AzureBatchFileConventions.IntegrationTests Batch\DataPlane\Azure.Batch.Unit.Tests Batch\DataPlane\Tools\ConfigureAwaitAnalyzer\ConfigureAwaitAnalyzer.Tests ContainerService\ContainerService.Tests PolicyInsights\Management\PolicyInsights.Tests Consumption\Consumption.Tests Billing\Billing.Tests ApplicationInsights\Management\ApplicationInsights.Tests Batch\Management\Management.Batch.Tests SecurityCenter\SecurityCenter.Tests AzureStack\Admin\FabricAdmin\Fabric.Admin.Tests IotHub\IotHub.Tests Monitor\Monitor.Tests"
+  - task: PublishTestResults@2
+    condition: succeededOrFailed()
+    inputs:
+      testResultsFiles: '**/*.trx'
+      testRunTitle: '$(OSName) DotNet $(DotNetCoreVersion)'
```

Comment:
You probably want to update the title to match what results you are seeing. The OSName and DotNetCoreVersion were variables that were defined the other yml file so the won't have any values for you here.